docs/administrator-documentation/moderne-platform/references/multi-tenant-vs-single-tenant.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: Single-tenant vs. Multi-tenant
+description: Discussing the various editions available in the Moderne Platform.
+---
+
+# Single-tenant vs. Multi-tenant
+
+The Moderne Platform offers two editions based on your needs: multi-tenant and single-tenant. Both editions provide full, unlimited access to all of the platform's automated, distributed code remediation and migration features. They both also include advanced code impact analysis, code intelligence dashboards, and AI integrations.
+
+The editions are differentiated by their implementations, enabling you to choose what fits the needs of your organization.
+
+In this doc, we'll walk through the differences between the two. By the end, you should have a clear idea of which one is suitable for your company.
+
+## Single-tenant
+
+The single-tenant edition of Moderne is one where your company receives a dedicated, fully isolated instance of the Moderne Platform. The cloud provider and region for this instance can be configured based on your preferences.
+
+For details about the single-tenant edition, please see the following docs:
+
+* [Moderne Platform architecture](./architecture.md)
+* [Authentication in the Moderne Platform](./authentication.md)
+* [How sensitive data flows through the Moderne Platform](./flow.md)
+* [What reports are available in the Moderne Platform](./reporting.md)
+
+## Multi-tenant
+
+The multi-tenant edition of Moderne is one where multiple companies have a secure, private area in a shared cloud instance of the Moderne Platform. Since multiple companies share a single cloud instance and resources, your contract with Moderne will cost less than it would if your company had their [own dedicated instance](#single-tenant).
```

Comment:
How about this?

The Standard deployment model offers a more cost-effective operational footprint by leveraging shared infrastructure, allowing it to be provided at a lower price point. In contrast, Enterprise deployments are typically more expensive, as they operate on dedicated resources and do not share infrastructure with other customers.